pkg/crc/cluster/cluster.go

```diff
@@ -223,3 +223,15 @@ func DeleteOpenshiftApiServerPods(oc oc.OcConfig) error {
 
 	return errors.RetryAfter(60, deleteOpenshiftApiserverPods, time.Second)
 }
+
+func CheckProxyIsPartOfAuthOperator(oc oc.OcConfig, proxy *network.ProxyConfig) (bool, error) {
+	cmdArgs := []string{"set", "env", "deployment", "redhat-operators", "--list", "-n", "openshift-marketplace"}
```

Comment:
Maybe it's worth starting the method with `if !proxy.IsEnabled() { return true, nil }`, potentially with a `logging.Debugf` to know what's going on?